### PR TITLE
Added badges to README

### PR DESCRIPTION
**Pull request by sb43**

### CODE REVIEW

**Comment by sb43** [comment]
@keiranmraine  travis badge in README is not displayed as intended

File: README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+
 Calculates the Variant Allele Fraction for variants sites in VCF and/or BED file 
 This script performs comparative analysis of variant sites in multiple tumour/normal samples in an individual.
 Also facilitates the merging of varinats sites across the samples in a sample group defined by 
 set of related samples in an individual and provides unbiased pileup[MNV] and exonerate[Indel] output for each variant site.
 
-| Master                                        | Develop                                         |
-| --------------------------------------------- | ----------------------------------------------- |
-| [![Master Badge][travis-master]][travis-base] | [![Develop Badge][travis-develop]][travis-base] |
-
```
Comment (keiranmraine) [inline README.md]: You need to add the links `[travis-***]` is a defined url at the very bottom of the example I sent you

File: setup.sh [hunk 2]
```diff
@@ -266,9 +266,9 @@ echo -n "Building exonerate..."
 (
   if [ -e $SETUP_DIR/$CURR_TOOL.success ]; then
     echo -n " previously installed ..."
-  else [ $INST_METHOD -eq 2 ]; then
+  elif [ $INST_METHOD -eq 2 ]; then
 		echo " Skipping exonerate install ..."
```
Comment (keiranmraine) [inline setup.sh]: Hard tabs!!!, you should be using 2 space soft tabs in all file formats (not Makefile)

File: setup.sh [hunk 3]
```diff
@@ -266,9 +266,9 @@ echo -n "Building exonerate..."
 (
   if [ -e $SETUP_DIR/$CURR_TOOL.success ]; then
     echo -n " previously installed ..."
-  else [ $INST_METHOD -eq 2 ]; then
+  elif [ $INST_METHOD -eq 2 ]; then
```
Comment (keiranmraine) [inline setup.sh]: Why `2`? You don't use `1` anywhere.  Please don't copy/paste without evaluating the intent.

File: CHANGES.md [hunk 4]
```diff
@@ -1,4 +1,7 @@
 
+## v5.3.4 #######
```
Comment (keiranmraine) [inline CHANGES.md]: I recommend you use a markdown linter, this is not valid heading syntax, it may render but these are unnecessary trailing characters.

**Comment by sb43** [comment]
Corrected suggested changes, build passed in ~5min

File: README.md (hunk 1 above)
Comment (sb43) [reply README.md]: Thanks, done now

File: setup.sh (hunk 2 above)
Comment (sb43) [reply setup.sh]: sorry should have been looked more carefully

File: setup.sh (hunk 3 above)
Comment (sb43) [reply setup.sh]: sorry should have been looked more carefully

File: setup.sh (hunk 2 above)
Comment (sb43) [reply setup.sh]: removed hard tabs

File: README.md [hunk 5]
```diff
@@ -1,20 +1,96 @@
...
+5. Commit the updated docs tree and updated module/version.
+6. Push commits.
+7. Use the GitHub tools to draft a release.
 
-# LICENCE
+## LICENCE
 
-Copyright (c) 2017 Genome Research Ltd.
+```
+Copyright (c) 2014-2018 Genome Research Ltd.
```
Comment (keiranmraine) [inline README.md]: You can't change the date to 2014-2018 if the code was only created in 2017

File: perl/lib/Sanger/CGP/Vaf.pm [hunk 6]
```diff
@@ -2,7 +2,7 @@ package Sanger::CGP::Vaf;
 use strict;
 use Const::Fast qw(const);
 
-our $VERSION = '5.3.2';
+our $VERSION = '5.3.4';
```
Comment (keiranmraine) [inline perl/lib/Sanger/CGP/Vaf.pm]: You need to delete the tag for 5.3.3 from github if the version was not updated.

File: README.md (hunk 5 above)
Comment (keiranmraine) [inline README.md]: You can't change the date to 2014-2018 if the code was only created in 2017

**Pull request closed without merging**